src/Administration/Resources/app/administration/src/module/sw-extension/component/sw-extension-card-base/sw-extension-card-base.html.twig

```diff
@@ -9,6 +9,15 @@
     <sw-loader v-if="isLoading" />
     {% endblock %}
 
+    <!-- eslint-disable-next-line sw-deprecation-rules/no-twigjs-blocks -->
+    {% block sw_extension_card_base_type_icon %}
```

Comment:
you should not add twig blocks in new code ;)

Reply:
Why not? Anyone could could want and add a custom column in there

Reply:
Done